### PR TITLE
Closes #444  | Create dataset loader for Sarawak Malay

### LINKED ISSUE #444: Create dataset loader for Sarawak Malay
Dataloader name: `sarawak_malay/sarawak_malay.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?sarawak_malay

|     Dataset| sarawak_malay |
|-------------|---|
| Description | This is a Sarawak Malay conversation data for the purpose of speech technology research. At the moment, this is an experimental data and currently used for investigating speaker diarization. The data was collected by Faculty of Computer Science and Information Technology, Universiti Malaysia Sarawak.  The data consists of 38 conversations that have been transcribed using Transcriber (see TextGrid folder), where each file contains two speakers. Each conversation was recorded by different individuals using microphones from mobile devices or laptops thus, different file formats were collected from the data collectors. All data was then standardized to mono, 16000Khz, wav format. |
| Subsets | - |
| Languages | zlm |
| Tasks | Text-To-Speech Synthesis, Automatic Speech Recognition |
| License | Creative Commons Zero v1.0 Universal (cc0-1.0) |
| Homepage | https://github.com/sarahjuan/sarawakmalay |
| HF URL | - |
| Paper URL | - |

### PR DESCRIPTION
**Pull request by djanibekov**
Closes #444 

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

File: seacrowd/sea_datasets/sarawak_malay/sarawak_malay.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+            audio_file = audiosegment.from_file(f"{filepath}/wav/{filename}.wav").resample(sample_rate_Hz=16000)
+
+            for sarawak_tg, malay_tg, speaker in zip(sarawak_conversation, malay_conversation, speakers):
+                start, end, text = sarawak_tg.minTime, sarawak_tg.maxTime, sarawak_tg.mark
+                malay_text = malay_tg.mark
+                speaker_id = speaker.mark
+
+                # if text == '':
+                #     continue
```
Comment (MJonibek) [inline seacrowd/sea_datasets/sarawak_malay/sarawak_malay.py:159]:
```suggestion
 
```

**Review by MJonibek — APPROVED** [review]
@djanibekov checked, LGTM. Only one small thing to fix: can you remove these comments

**Comment by djanibekov** [comment]
@MJonibek , done

**Review by yongzx — APPROVED** [review]
LGTM. Tested with `python3 -m tests.test_seacrowd`

**Comment by yongzx** [comment]
Merging this now.

**Pull request merged**